.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# See the code-owners Github docs:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Mostly backend-y directories
+/bedrock @mozilla/bedrock-codeowners-backend
+/docker @mozilla/bedrock-codeowners-backend
+/lib @mozilla/bedrock-codeowners-backend
+/requirements @mozilla/bedrock-codeowners-backend
+/wsgi @mozilla/bedrock-codeowners-backend
+Dockerfile @mozilla/bedrock-codeowners-backend
```

Comment:
The most frequent bumps in docker builder might be node images version; that's for FE to review perhaps?